Modules/CustomRolesHelper.cs

```diff
@@ -1287,11 +1302,10 @@ public static CountTypes GetCountTypes(this CustomRoles role)
             CustomRoles.Pickpocket => CustomWinner.Pickpocket,
             CustomRoles.Traitor => CustomWinner.Traitor,
             CustomRoles.Vulture => CustomWinner.Vulture,
-            CustomRoles.Pestilence => CustomWinner.Pestilence,
+            CustomRoles.Apocalypse => CustomWinner.Apocalypse,
```

Comment:
Maybe here need use
`var role when role.IsNA() => CustomWinner.Apocalypse,`
And remove `CustomRoles.Apocalypse` because this unused?

Reply:
hmm wait